core/src/UserSettingsDialog.html

```diff
@@ -537,6 +472,16 @@
       z-index: 1;
     }
   }
+
+  /*need to be removed after updating FD Styles to 0.16.0, selected state for left side nav*/
```

Comment:
```suggestion
  /* TODO: need to be removed after updating FD Styles to 0.16.0, selected state for left side nav */
```
just a suggestion. makes it easier to search later on.